src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1764,13 +1764,27 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         .setProcessingSpeed(1)
         .register(plugin);
 
+        new AutoEnchanter(categories.electricity, SlimefunItems.AUTO_ENCHANTER_2, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.REINFORCED_PLATE, SlimefunItems.BIG_CAPACITOR, SlimefunItems.REINFORCED_PLATE, SlimefunItems.BLISTERING_INGOT_3, SlimefunItems.AUTO_ENCHANTER, SlimefunItems.BLISTERING_INGOT_3, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.SYNTHETIC_DIAMOND, SlimefunItems.WITHER_PROOF_OBSIDIAN})
+        .setCapacity(1024)
+        .setEnergyConsumption(15)
```

Comment:
You set the power per sec to 48, this should be 24 then.

Reply:
uhhh i tested it with 30J/s i forgot to change those